CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.0] - 2020-06-29
```

Comment:
This does not get added until you make a release, so for now it should be under unreleased.

Once a release is made or you feel confident that this in a working state then move those lines to be under the new version with the date, like you already this.

Example:
before release

## [Unreleased]

- added controllers.
- added routes.
- ...

after realease

## [Unreleased]

## [1.0.0] - YYYY-MM-DD

- added controllers.
- added routes.
- ...